h/static/scripts/root-thread.js

```diff
@@ -65,6 +65,8 @@ function RootThread($rootScope, annotationUI, features, searchFilter, viewFilter
           return thread.annotation && metadata.isAnnotation(thread.annotation);
         } else if (state.selectedTab === uiConstants.TAB_NOTES) {
           return thread.annotation && metadata.isPageNote(thread.annotation);
+        } else {
+          return true;
```

Comment:
Strictly speaking this is a change from current behaviour (currently this function would return a falsy value), but as long as we currently never hit this branch I guess this is the right fallthrough.